src/scripting.rs

```diff
@@ -0,0 +1,15 @@
+use bevy::prelude::*;
+use bevy_mod_js_scripting::{JsRuntimeConfig, JsScriptingPlugin};
+
+mod ops;
+
+pub struct ScriptingPlugin;
+
+impl Plugin for ScriptingPlugin {
+    fn build(&self, app: &mut App) {
+        let custom_ops = ops::get_ops();
+
+        app.insert_non_send_resource(JsRuntimeConfig { custom_ops })
```

Comment:
What does it mean that its non send? Only accessible from main thread?

Reply:
Yes. Since `JsRuntimeConfig` contains structs that may not implement `Send`, bevy has to make sure it only gets inserted and accessed on the main thread.